docs/report_stage_iii.md

```diff
@@ -242,17 +253,25 @@ Sample results:
 
 <img src="./report_stage_iii_assets/besu_arguments_exp_arg1.png" width="700"/>&nbsp;
 
-*Analysis*
+#### Analysis
 
 The timings for Besu are characterised by a rather significant scatter, even after removing outliers. This might be due to the JVM garbage collector, which is not controlled in our setup. Additionally, the engine overhead is rather large. We tried to remove it from the results, but the results were not consistent.
 
 As seen on the charts, the execution times for `ADD`, `DIV` and `MULMOD` are not consistent. While the majority behaves in a linear fashion, there is a large portion that often takes longer. Again, we attribute it to the JVM garbage collector or other JVM internals.
 
-The execution time for `EXP` increases linearly with the size of the second argument. Again the separation into two different lines shows that the execution time depends not only on the size of the argument but also its value. In contrast to other implementations, the second line has a constant slope.
+The execution time for `EXP` increases linearly with the size of the second argument. Again the separation into two different lines shows that the execution time depends not only on the size of the argument but also on its value. In contrast to other implementations, the second line has a constant slope.
+
+Some common OPCODEs perform worse than expected. Improving them would have a positive impact on the overall execution time. These are `LT`, `GT`, `AND`, `OR`, `XOR` and `BYTE`.
```

Comment:
I would avoid the terms "worse", "improve performance". Improving the execution time is not our goal. The fact that estimations for these OPCODEs are relatively higher can mean that there is a space for optimizations or other OPCODEs are very efficiently implemented.